### PR TITLE
qr code page

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers 
Do 'already on keybase? login'
Connect another device
Phone

You'll see 
screen shot 2016-02-22 at 1 46 15 pm

### CODE REVIEW

File: shared/common-adapters/text.desktop.js [hunk 1]
```diff
@@ -50,6 +50,10 @@ export default class Text extends Component {
         return {color: backgroundMode === 'Normal' ? globalColorsDZ2.black75 : globalColorsDZ2.white}
       case 'BodySmall':
         return {color: backgroundMode === 'Normal' ? globalColorsDZ2.black40 : globalColorsDZ2.white40}
+      case 'BodyPrimaryLink':
```
Comment (chrisnojima) [inline shared/common-adapters/text.desktop.js]: we just have standalone link styles now, so no additional link paramter

File: shared/constants/login.js [hunk 2]
```diff
@@ -8,7 +8,6 @@ export const codePageModeScanCode = 'codePageModeScanCode'
 export const codePageModeShowCode = 'codePageModeShowCode'
 export const codePageModeEnterText = 'codePageModeEnterText'
 export const codePageModeShowText = 'codePageModeShowText'
-export const codePageModeShowCodeOrEnterText = 'codePageModeShowCodeOrEnterText'
```
Comment (chrisnojima) [inline shared/constants/login.js]: we don't have this screen anymore with the combined showcode+entertext

**Comment by chrisnojima** [comment]
@keybase/react-hackers  bump

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**